### ISSUE TITLE
How to implement a detach operation similar to Pytorch?

### ISSUE THREAD

**Issue by zaoanhh** [LuxDL/Lux.jl#1138]
I've been doing some work recently using Lux for unsupervised training, and I think Lux is pretty cool. But I encountered some problems: I used to use the output of the model to generate some labels for subsequent calculations. In Pytorch it looks like:
```python
import torch
import torch.nn.functional as F
y = model(x)
y1 = y.detach()
softmax_y1 = F.softmax(y1, dim=1)
pred_class_indices = torch.argmax(softmax_y1, dim=1)
num_classes = 4
labels_true = F.one_hot(pred_class_indices, num_classes=num_classes)
```
In Lux, how should I implement the detach operation to avoid the gradient being tracked during label generation?

**Comment by avik-pal** [comment]
You do `ChainRulesCore.ignore_derivatives(y)`

**Comment by zaoanhh** [comment]
Thank you!

**Comment by zaoanhh** [comment]
> You do `ChainRulesCore.ignore_derivatives(y)`

Thank you!
